pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -160,12 +161,12 @@ class Command(BaseCommand):
                     default='',
                     help="Target language to fallback to use in case it can't "
                          "be guessed for any of the input files."),
-        make_option('--project',
+        make_option('--display-name',
                     action='store',
                     dest='project',
                     default='',
-                    help='Project to use when displaying TM matches for this '
-                         'translations.'),
+                    help='Name to use as provenance when displaying TM '
+                         'matches for these translations.'),
```

Comment:
You like the word provenance :)

Name used when displaying TM matches for these translations.